docs/templates/modules.md

```diff
@@ -29,32 +29,32 @@ resource "coder_agent" "dev" {
 }
 ```
 
-> Learn more about
-> [creating modules](https://developer.hashicorp.com/terraform/language/modules)
-> and
-> [module sources](https://developer.hashicorp.com/terraform/language/modules/sources)
-> in the Terraform documentation.
+Learn more about
+[creating modules](https://developer.hashicorp.com/terraform/language/modules)
+and
+[module sources](https://developer.hashicorp.com/terraform/language/modules/sources)
+in the Terraform documentation.
 
 ## Git authentication
 
-If you are importing a module from a private git repository, the Coder server
-[or provisioner](../admin/provisioners.md) needs git credentials. Since this
```

Comment:
btw, in `docs/manifest.json`, the entry for `docs/admin/provisioners.md` should have this title:

```
          "title": "External provisioners",
```